app/views/faq.scala.html

```diff
@@ -123,46 +127,47 @@ <h3 class="govuk-heading-s">Can I share my sign-in information?</h3>
             </p>
 
             <h3 class="govuk-heading-s">I cannot choose my series reference</h3>
             <p class="govuk-body">
-                If you cannot find your series reference in the dropdown list on the ‘Series reference’ page when you begin the transfer process, contact <a href="mailto:@Messages("nationalArchives.email")">
+                If you cannot find your series reference in the dropdown list on the ‘Series reference’ page, or you are unsure what your series reference should be when you begin the transfer process, contact <a href="mailto:@Messages("nationalArchives.email")">
                 @Messages("nationalArchives.email")</a>.
             </p>
 
             <h3 class="govuk-heading-s">What is a consignment reference?</h3>
             <p class="govuk-body">
-                A consignment reference is a unique label assigned to each consignment (batch) of digital records that have been uploaded to TDR. Your dashboard and transfer summary will show you your consignment reference. It will also be displayed in the top right-hand corner of your screen once you have begun a transfer.
+                A consignment reference is a unique label assigned to each consignment (batch) of digital records that have been uploaded to TDR, for example 'MOCK-123-TDR'. The View Transfers page will show you your consignment reference. It will also be displayed in the top right-hand corner of your screen once you have begun a transfer.
             </p>
 
             <h3 class="govuk-heading-s">What are Public Records?</h3>
             <p class="govuk-body">
                 Public Records are records created or held by any UK government department or related body. The National Archives can usually only accept Public Records.
             </p>
 
-            <h3 class="govuk-heading-s">How do I know if my records are Crown copyright?</h3>
+            <h3 class="govuk-heading-s">How do I know if my records are Crown Copyright?</h3>
             <p class="govuk-body">
-                Works created by officers or servants of the crown during the course of their duties are Crown copyright. This includes material created by ministers, civil servants, government departments and agencies. The National Archives will accept Public Records that are not Crown copyright, however during this initial private beta stage, you cannot currently use TDR to transfer them to us.
+                Works created by officers or servants of the crown during the course of their duties are Crown Copyright. This includes material created by ministers, civil servants, government departments and agencies. In the future The National Archives will accept Public Records that are not Crown Copyright, however during this initial development stage, you cannot use TDR to transfer them to us.
```

Comment:
The word document has got "officers or servants of the **C**rown" if I'm reading it right and copyright is all lower case in the heading and paragraph.

Reply:
There was inconsistencies with the capitalisation of the "Crown" and "Copyright", so its consistent now.